### PR TITLE
adds genrule for packaging debian

### PR DESCRIPTION
**Pull request by nikhedonia**
#1465 adds genrule to create a debian package.
TODO: Edit Changelog
TODO: Automate generation of bucks version number in debian/buck.equivs
TODO: bundle with openjdk (see https://github.com/LoopPerfect/buckaroo/blob/master/debian/buckaroo-with-jdk.equivs)

### CODE REVIEW

File: programs/BUCK [hunk 1]
```diff
@@ -25,6 +25,7 @@ python_binary(
     deps = [
         ":buck_version",
     ],
+    visibility = ['PUBLIC'],
```
Comment (ttsugriy) [inline programs/BUCK]: please use a more specific include here and below.

File: BUCK [hunk 2]
```diff
@@ -0,0 +1,22 @@
+genrule(
```
Comment (ttsugriy) [inline BUCK]: please move this and everything relating to debian packaging into `scripts/packages/debian`

Comment (nikhedonia) [inline BUCK]: how can I access files in the project-root like LICENSE from scripts/packages/debian ?

Comment (ttsugriy) [inline BUCK]: @nikhedonia, you should be able to use [export_file](https://buckbuild.com/rule/export_file.html) to export license file and add a dependency on it just like on any other rule.

**Pull request closed without merging**